astropy/coordinates/earth.py

```diff
@@ -555,12 +571,11 @@ def _get_site_registry(cls, force_download=False, force_builtin=False):
                     if force_download:
                         raise
                     msg = (
-                        "Could not access the online site list. Falling "
-                        "back on the built-in version, which is rather "
-                        "limited. If you want to retry the download, do "
-                        "{0}._get_site_registry(force_download=True)"
+                        "Could not access the main site list. Falling back on the "
+                        "built-in version, which is rather limited. If you want to "
+                        "retry the download, use the option `refresh_cache=True`."
                     )
-                    warn(AstropyUserWarning(msg.format(cls.__name__)))
+                    warn(AstropyUserWarning(msg))
```

Comment:
While we're at this, isn't this a more proper usage?
```suggestion
                    warn(msg, AstropyUserWarning)
```

Reply:
Yes indeed. Furthermore, this warning isn't being tested. @tbowers7, are you interested in applying the suggestion by @pllim and adding a unit test for this or do you think this is too far beyond the scope of your original idea?

Reply:
To be sure, you can generate the coverage report locally when you test this branch and enable `--remote-data` pytest flag.

Reply:
This warning is emitted if accessing the remote server fails, so there could and should be a test for this that doesn't require remote data.

Reply:
It sounds like a unit test for this warning should be created, but I would suggest this be left as an [issue](https://github.com/astropy/astropy/issues) separate from this PR.  I would, however, be willing to entertain a friendly commit that implements this unit test from someone more familiar with the intricacies of AstroPy's testing structures.

Reply:
I don't think this warning should hold back this pull request, I'd rather have a follow-up pull request add the missing test.